proposals/jib_core_library.md

```diff
@@ -52,25 +73,45 @@ Three `TargetImage` types (`RegistryImage`, `DockerDaemonImage`, and `TarImage`)
 - `Containerizer setCacheConfiguration(CacheConfiguration)`
 - `Containerizer setEventHandlers(EventHandlers)`
 
-## For Java containers
+## Simple example
+
+```java
+Jib.from("busybox")   
+   .addLayer(Arrays.asList(Paths.get("helloworld.sh")), "/") 
+   .setEntrypoint("/helloworld.sh")
+   .containerize(
+       Containerizer.to(RegistryImage.named("coollog/jibtestimage")
+                           .setCredential("coollog", "notmyrealpassword")));
+```
+
+# For Java containers
+
+`JavaContainerBuilder` - builds a `JibContainerBuilder` for Java apps. 
 
-`JavaContainerBuilder` - builds a `JibContainerBuilder` for Java apps
 - `static JavaContainerBuilder builder()`
 - `JavaContainerBuilder addDependencies(List<Path> dependencyFiles)`
 - `JavaContainerBuilder addResources(List<Path> resourceFiles)`
 - `JavaContainerBuilder addClasses(List<Path> classFiles)`
 - `JavaContainerBuilder addToClasspath(List<Path> otherFiles)`
 - `JavaContainerBuilder setJvmFlags(List<String>/String... jvmFlags)`
 - `JavaContainerBuilder setMainClass(String mainClass)`
-- `JavaContainerBuilder toContainerBuilder()`
+- `JibContainerBuilder toContainerBuilder()`
 
-# Simple example
+  Returns a `JibContainerBuilder` that, when built, will produce a minimal Java image that executes the java application specified by the builder methods previously called on this `JavaContainerBuilder`. 
 
-```java
-Jib.from("busybox")
-   .addLayer(Arrays.asList(Paths.get("helloworld.sh")), "/helloworld.sh")
-   .setEntrypoint("/helloworld.sh")
-   .containerize(
-       Jib.to(RegistryImage.named("coollog/jibtestimage")
-                           .setCredential("coollog", "notmyrealpassword"));
-```
+  Example:
+
+  ```java
+  	RegistryImage destination = RegistryImage.named("gcr.io/myuser/my-java-container:latest");
+
+    JibContainerBuilder javaImage = JavaContainerBuilder.builder()
```

Comment:
`JavaContainerBuidler javaImage = ...`, since `JibContainerBuilder` doesn't have `addDependencies()` and such. And rename `javaImage` to `javaContainerBuilder` or `containerBuilder`, since the builder is not an `Image`.

 @GoogleContainerTools/java-tools I don't remember, will we have `JavaContainerBuilder` and methods like `addDependencies`, etc?